sonic_installer/bootloader/bootloader.py

```diff
@@ -75,6 +75,10 @@ def supports_package_migration(self, image):
         """tells if the image supports package migration"""
         return True
 
+    def verify_image_sign(self, image_path):
```

Comment:
>verify_image_sign [](http://example.com/codeflow?start=8&length=17)

A unsecure normal image is considered not signed image, and should return False.

Reply:
We want to verify image signature only of secure boot flag is enabled. This flow was already merged in previous PR  https://github.com/sonic-net/sonic-utilities/pull/2337 (that was reverted even though it had a fix ready in a day) and described and approved in the HLD as well (https://github.com/sonic-net/SONiC/pull/1024)

Reply:
@qiluo-msft please add your comments/approval

Reply:
After checking HLD, my comment is still applicable.

Reply:
If you mean "image verification not implemented", you should either throw NotImplmented exception or return a reasonable value to indicate the error. Silently "continue without verification" does not make sense.

Reply:
Modified the code as suggested, please review @qiluo-msft